File: 0058-length-of-last-word/0058-length-of-last-word.cpp
int su = []{std::ios::sync_with_stdio(0); std::cin.tie(0); return 0; }();
class Solution {
public:
    int lengthOfLastWord(string s) {
        su;
        int c=0;
        int n=s.length()-1;
        while(s[n]==' ')
            n--;
        while(n>=0){
            if(s[n]!=' ')
            {
                c++;
                n--;
            }
            else
            break;
        }
        return c;
    }
};

File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        s = s.strip() 
        l= 0
    
        for i in range(len(s) - 1, -1, -1):
            if s[i] == ' ':
                break
            l+= 1
    
        return l